CMakeLists.txt

```diff
@@ -6,13 +6,17 @@ project(picoprobe)
 
 pico_sdk_init()
 
+# Comment out to use Pico LED
+add_definitions(-DUSE_WS2812)
```

Comment:
This should probably be something that's off-by-default that you then need to turn on for the Adafruit QTPy2040, rather than something that's on-by-default that you then need to turn off for the Raspberry Pi Pico? :wink:

Reply:
That seems very reasonable